apps/reference/docs/guides/api.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: api
-title: APIs
+title: Auto-generated APIs
```

Comment:
I discussed with Ant - perhaps we go with "Data APIs" (similar to Aurora) and then we use "Auto-generated" just in the tagline

For the Admin API, perhaps we go with "Management API"

Thoughts?

Reply:
Works for me!